app/states/unified_field_library.py

```
@@ -0,0 +1,503 @@
+                    for mapping in mappings
+                ]
+                for state, mappings in self.state_mappings.items()
+            }
+        }
+        
+        with open(file_path, 'w') as f:
+            json.dump(export_data, f, indent=2)
+    
+    def import_mappings(self, file_path: Path):
+        """Import mappings from a JSON file"""
+        with open(file_path, 'r') as f:
```

Ruff (0.14.7)

Unnecessary mode argument

Remove mode argument

(UP015)